test/features/composer/presentation/model/saved_email_draft_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+import 'package:tmail_ui_user/features/composer/presentation/model/saved_email_draft.dart';
+
+void main() {
+  group('saved email draft test', () {
+    test(
+      'should tag toRecipients, ccRecipients and bccRecipients in props',
+    () {
+      // arrange
+      final savedEmailDraft = SavedEmailDraft(
+        subject: 'subject',
+        content: 'content',
+        toRecipients: {EmailAddress('to name', 'to email')},
+        ccRecipients: {EmailAddress('cc name', 'cc email')},
+        bccRecipients: {EmailAddress('bcc name', 'bcc email')},
+        identity: null,
+        attachments: [],
+        hasReadReceipt: false
+      );
+      
+      // act
+      final props = savedEmailDraft.props;
+      
+      // assert
+      expect(props[2], equals({0: savedEmailDraft.toRecipients}));
+      expect(props[3], equals({1: savedEmailDraft.ccRecipients}));
+      expect(props[4], equals({2: savedEmailDraft.bccRecipients}));
```

Comment:
Should add unit test to check if `hashCode` is different when having same email address in `to`, `cc`, `bcc`

Reply:
Done